MdePkg/Include/IndustryStandard/Nvme.h

```diff
@@ -663,6 +708,9 @@ typedef struct {
   UINT32    Pi    : 3;        /* Protection Information */
   UINT32    Pil   : 1;        /* Protection Information Location */
   UINT32    Ses   : 3;        /* Secure Erase Settings */
+  #define SES_NO_SECURE_ERASE  0x0
+  #define SES_USER_DATA_ERASE  0x1
+  #define SES_CRYPTO_ERASE     0x2

```

Comment:
I was a bit unsure here which style to follow. The nvme.h file has several instances where #defines are declared right inside the structure (see below examples). So I was trying to stick to precedent. However, if Mu uses a different convention, I'm fine moving the #defines outside the structure definition. 

typedef struct {
  //
  // CDW 10
  //
  UINT32    Lid   : 8;        /* Log Page Identifier */
  #define LID_ERROR_INFO    0x1
  #define LID_SMART_INFO    0x2
  #define LID_FW_SLOT_INFO  0x3
  #define LID_BP_INFO       0x15
  UINT32    Rsvd1 : 8;
  UINT32    Numd  : 12;       /* Number of Dwords */
  UINT32    Rsvd2 : 4;        /* Reserved as of Nvm Express 1.1 Spec */
} NVME_ADMIN_GET_LOG_PAGE;

Reply:
I saw that after I left the comment. I think those are violating the normal convention as well. Since it's already done that way here elsewhere, I am fine with it being left as-is.